package.json

```diff
@@ -26,30 +29,60 @@
     "@kunigi/string-compression": "1.0.2",
     "@mdx-js/loader": "^2.3.0",
     "@mdx-js/react": "^2.3.0",
+    "@monaco-editor/react": "^4.6.0",
```

Comment:
You added like 20 new dependencies which is something that usually needs to be discussed and agreed on.

You are mixing different styling approaches like MUI, RadixUI, TailwindCSS in one project which will blow up the bundle size.